src/diagnostics/Diagnostics.jl

```diff
@@ -45,17 +45,18 @@ import ..compute_gm_mixing_length!
 include(joinpath("..", "utils", "abbreviations.jl"))
 
 import ClimaDiagnostics:
-    DiagnosticVariable,
-    ScheduledDiagnostic,
-    average_pre_output_hook!,
-    DiagnosticsCallback
+    DiagnosticVariable, ScheduledDiagnostic, average_pre_output_hook!
 
 import ClimaDiagnostics.DiagnosticVariables: descriptive_short_name
 
 import ClimaDiagnostics.Schedules: EveryStepSchedule, EveryDtSchedule
 
-
-import ClimaDiagnostics.Writers: HDF5Writer, NetCDFWriter, write_field!
+import ClimaDiagnostics.Writers:
+    HDF5Writer,
+    NetCDFWriter,
+    write_field!,
+    LevelsMethod,
+    FakePressureLevelsMethod
```

Comment:
I guess the `Fake` part here is the approximation/ assumptions when interpolating to pressure levels? Can you please also summarise the new features users can expect to the `news.md` file (or include a link to the relevant ClimaDiagnostics documentation)? Thanks!

Reply:
There are no new features or changes for users: it's a completely internal change.

Yes, the `Fake` is in the pressure levels use a very simplified model.